src/components/VSketchFabViewer.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="block max-w-[1000px] max-h-[500px] aspect-[2/1]">
+    <iframe
+      id="sketchfab-iframe"
+      ref="node"
+      src=""
+      sandbox="allow-scripts allow-same-origin allow-popups allow-forms"
+      title="Sketchfab iframe"
```

Comment:
Can we use i18n here? I've read today that there's inconsistency in the a11y tools, so it is advised to use both `title` and `aria-label` for iframes.

Reply:
Oops, yes, definitely! I will make that change.